### ISSUE TITLE
Question about the abstract class GraspModel

### ISSUE THREAD

**Issue by ShiJiajie0805** [skumra/robotic-grasping#29]
pos_pred, cos_pred, sin_pred, width_pred = self(xc)

I am a python new guy
Why this code which is in the abstract class GraspModel could call the method forward in the subclass GenerativeResnet

**Comment by skumra** [comment]
Yeah, probably not the best implementation. Please feel free to open a pull request to refactor it.

**Comment by chenjiadragon** [comment]
> pos_pred, cos_pred, sin_pred, width_pred = self(xc)
> 
> I am a python new guy Why this code which is in the abstract class GraspModel could call the method forward in the subclass GenerativeResnet
 
Thank you for telling me that this step is forward propagation
